Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -40,6 +41,12 @@ struct GenerateCommand: AsyncParsableCommand, HasTrackableParameters {
     )
     var xcframeworks: Bool = false
 
+    @Option(
+        name: [.long],
+        help: "Type of cached xcframeworks to use when --xcframeworks is passed (device/simulator)"
+    )
+    var destination: CacheXCFrameworkDestination?
```

Comment:
It's weird that we are not reusing `CacheOptions`, but it was already like this and I can try to check in a separate PR 🙏